packages/cli/test/initializer/truffle/Truffle.test.js

```diff
@@ -84,6 +84,24 @@ contract('Truffle', () => {
     })
   })
 
+  describe('validateAndLoadNetworkConfig', function () {
+    context('when the requested network does not exist in the truffle config file', function () {
+      context('when truffle config file is truffle.js', function () {
+        it('throws an error', function () {
+          FileSystem.write(`${testDir}/truffle.js`, 'module.exports = { networks: { test: { gas: 1, gasPrice: 2, from: \'0x0\' } } }');
+          (() => Truffle.validateAndLoadNetworkConfig('non-existent', true, testDir)).should.throw(/is not defined in your truffle.js file/);
```

Comment:
Should we clear out the written file after the spec?

Reply:
The whole folder is being remove after each spec (see [L13](https://github.com/zeppelinos/zos/pull/710/files#diff-5f3f7b3c301cfeba822f41b337060c23R13))